File: CS/Sample/Controllers/HomeController.cs
using System.Web.Mvc;

namespace Sample.Controllers
{
    public class HomeController : Controller {

        public ActionResult Index() {
            return View();
        }
        public ActionResult GridViewPartial(bool? resizing) {
            ViewData["resizing"] = resizing;
            return PartialView();
        }
    }
}

File: VB/Sample_VB/Controllers/HomeController.vb
Namespace Controllers
    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            Return View()
        End Function
        Public Function GridViewPartial(ByVal resizing? As Boolean) As ActionResult
            ViewData("resizing") = resizing
            Return PartialView()
        End Function
    End Class
End Namespace